src/components/badge/badge-test.stories.tsx

```diff
@@ -33,9 +33,28 @@ export const DefaultStory = ({ counter, ...args }: BadgeStoryProps) => {
   );
 };
 
+export const DisplayOnlyStory = ({ counter, ...args }: BadgeStoryProps) => {
+  return (
+    <div style={{ margin: "40px" }}>
```

Comment:
We should avoid adding new `style` prop instances - https://github.com/Sage/carbon/pull/5869

Reply:
Same comment as previous. Completely forgot about that PR. I'll swap the `div` out for `Box`.